package.json

```diff
@@ -71,8 +71,8 @@
     "react-dom": "^18.2.0",
     "react-load-script": "^0.0.6",
     "react-redux": "^8.0.4",
-    "react-router": "^6.4.2",
-    "react-router-dom": "^6.4.2",
+    "react-router": "^6.7",
+    "react-router-dom": "^6.7",
```

Comment:
The new `useBlocker` hook was only introduced in 6.7 (I'll update the changelog as well to reflect the new dependency constraint)